cf/api/service_plan.go

```diff
@@ -43,6 +44,22 @@ func (repo CloudControllerServicePlanRepository) Update(servicePlan models.Servi
 	return repo.gateway.UpdateResource(repo.config.ApiEndpoint(), url, strings.NewReader(body))
 }
 
+func (repo CloudControllerServicePlanRepository) ListPlansFromManyServices(serviceGuids []string) (plans []models.ServicePlanFields, err error) {
```

Comment:
We try to avoid using inline return variable declarations (like plans is here) because it can be hella confusing. Would you guys be ok with changing this to just have a generic return type and then declare plans in the function?